packages/alert/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@availity/mui-alert",
+  "version": "0.0.0",
+  "description": "Availity MUI Alert Component - part of the @availity/element design system",
+  "keywords": [
+    "react",
+    "typescript",
+    "availity",
+    "alert",
+    "mui"
+  ],
+  "homepage": "https://availity.github.io/element/?path=/docs/components-alert-introduction--docs",
+  "bugs": {
+    "url": "https://github.com/Availity/element/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Availity/element.git",
+    "directory": "packages/alert"
+  },
+  "license": "MIT",
+  "author": "Roxanne Young <roxanne.young@availity.com>",
+  "browser": "./dist/index.js",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "exports": {
```

Comment:
The `exports` here broke claim status for some reason so I had to remove them from the other packages. I think we should continue to leave it out until we understand it more